mobile/mobile.go

```diff
@@ -92,6 +95,16 @@ type Mobile struct {
 	listener  *broadcast.Listener
 }
 
+// RepoPath returns the actual location of the configured repo
+func (conf InitConfig) RepoPath() string {
+	return path.Join(conf.BaseRepoPath, conf.Seed)
+}
+
+// RepoExists return whether or not the configured repo already exists
+func (conf InitConfig) RepoExists() bool {
+	return fsrepo.IsInitialized(conf.RepoPath())
+}
+
```

Comment:
Similar updates to the mobile module. I considered proxying through to the core version of `InitConfig` so that `path` and `fsrepo` packages wouldn't have to be imported in the mobile module, but it makes the mobile API nastier because these methods would have to return `(<value>, error)` instead of a single value. Not worth it.